main_test.go

```diff
@@ -43,6 +45,10 @@ func (f *ComponentTest) InitializeTestSuite(ctx *godog.TestSuiteContext) {
 
 func TestComponent(t *testing.T) {
 	if *componentFlag {
+		// discarding production logging to obtain cleaner reporting of component specifications and results
```

Comment:
Please remove these lines as log output is required.
You are thinking about form over function.
If in the future someone makes a change in the code and it breaks things, then they will need to see logs.
Reports are of no real consequence, people when checking a feature test run are only looking at the final code.
The final output at the end of all of the feature tests that looks similar to a Zero error code of:
ERRO[0065] 0
Is all that matters.

Reply:
Added a flag to make the logging output optional during test runs